src/pages/colour.js

```diff
@@ -0,0 +1,49 @@
+import Link from "gatsby-link";
+import React from "react";
+
+import Title from "../components/Title";
+import Palette from "../components/Palette";
+import HeadingTwo from "../components/HeadingTwo";
+import Paragraph from "../components/Paragraph";
+
+import "./index.css";
+
+const Pages = () => (
+  <div>
+    <Link className="Pages-home" to="/">{`Home`}</Link>
+    <div className="Pages-title">
+      <Title>{`Colour`}</Title>
+    </div>
+    <div className="Pages-paragraph">
+      <Paragraph>{`The colour palette is:`}</Paragraph>
+    </div>
+    <Palette />
+    <div className="Pages-paragraph">
+      <Paragraph
+      >{`These colours are accessible for text greater than 18px.`}</Paragraph>
+    </div>
+    <div className="Pages-paragraph">
+      <Paragraph
+      >{`For text smaller than this, use a 20% shade to maintain AA compliance.`}</Paragraph>
+    </div>
+    <div className="Pages-headingTwo">
+      <HeadingTwo>{`Suggested usage`}</HeadingTwo>
+    </div>
+    <div className="Pages-colour">
+      <div className="Pages-colourItem">
+        <strong>{`Red`}</strong>
```

Comment:
Like `Paragraph`, do we want a component for `strong`?

Reply:
`Paragraph` is a component because it's an element from the suggested sizes and type section of the design system. Here we're just using a standard HTML element to give semantics. If we need to style it we would give it the class `Pages-colourItemName`.